### PR TITLE
Syntax: use const

### PR DESCRIPTION
**Pull request by nkbt**
All demos are working well

### CODE REVIEW

File: Spring.jsx [hunk 1]
```diff
@@ -283,14 +283,16 @@ export let TransitionSpring = React.createClass({
       return;
     }
     this._rafID = requestAnimationFrame(() => {
-      let {currVals, currV, now} = this.state;
-      let {endValue, willEnter, willLeave} = this.props;
+      let {currVals, currV} = this.state;
```
Comment (nkbt) [inline Spring.jsx]: These are actually modified later

File: utils.js [hunk 2]
```diff
@@ -14,8 +14,8 @@ export function range(start, afterStop) {
     afterStop = start;
     start = 0;
   }
-  let ret = [];
-  for (var i = start; i < afterStop; i++) {
+  const ret = [];
+  for (let i = start; i < afterStop; i++) {
```
Comment (nkbt) [inline utils.js]: var!

Comment (chenglou) [inline utils.js]: lol, gonna update editor autocomplete...

**Comment by chenglou** [comment]
:+1:

**Pull request merged**